src/citrine/resources/project.py

```diff
@@ -199,6 +200,48 @@ def make_private(self,
         })
         return True
 
+    def list_members(self) -> List[User]:
+        """
+        List all of the users in the current project.
+
+        Returns
+        -------
+        List[User]
+            Dict with key 'users' containing an array of User objects
+            These are the members of the current project
+
+        """
+        return self.session.checked_get(self._path() + "/users")["users"]
+
+    def set_member(self, user_uid, role: str, actions: List[str]) -> bool:
+        """
+        Add a User to a Project with the given 'role' and 'actions'.
+
+        Returns
+        -------
+        Returns True if member successfully added
```

Comment:
In order to get picked up by Sphinx and turned into nice documentation, I believe this has to follow the specific format of
```
Returns
---------
type
    description of return
```

Reply:
Is there a way I can run a linter against that?

Reply:
also 👍

Reply:
The only way I know of is to build the docs locally and inspect them. AFAIK a linter won't catch it because this isn't breaking any linting rules, it just doesn't follow the specification imposed by the sphinx extension.

Reply:
This makes me think we need some sort of "contributing guidelines" doc. I'll make an issue.

Reply:
https://pypi.org/project/restructuredtext-lint/ I found this

Reply:
Ooh, that's worth looking into.